filecoin-proofs/src/api/util.rs

```diff
@@ -29,3 +29,9 @@ pub(crate) fn get_tree_size<D: Domain>(sector_size: SectorSize) -> usize {
 
     2 * elems - 1
 }
+
+pub(crate) fn get_tree_leafs<D: Domain>(sector_size: SectorSize) -> usize {
+    let sector_size = u64::from(sector_size);
+
+    sector_size as usize / std::mem::size_of::<D>()
```

Comment:
A `Domain` is a `Element`, so this could use `D::byte_len()` instead of the `size_of`

Reply:
Thanks, will update, just want to see if CI is happy, or if anything else first.